.github/workflows/build_and_test.yml

```diff
@@ -15,6 +15,8 @@ on:
     - master
     - release/*
     - test-please/*
+  schedule:
+    - cron: "15 0 * * 0"
```

Comment:
apparently, [scheduled workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) only run on the default branch, so I couldn't test the scheduling on this branch